test/cpp/aoti_inference/test.cpp

```diff
@@ -76,6 +76,31 @@ void test_aoti_script(const std::string& device) {
   }
 }
 
+void test_aoti_package_loader(
+    const std::string& device,
+    bool use_runtime_constant_folding) {
+  torch::NoGradGuard no_grad;
+
+  std::string data_path =
+      (std::filesystem::path(STRINGIZE(CMAKE_CURRENT_BINARY_DIR)) / "data.pt")
+           .string();
+  torch::jit::script::Module data_loader = torch::jit::load(data_path);
+  std::string suffix = use_runtime_constant_folding
+      ? device + "_use_runtime_constant_folding"
+      : device;
+  std::string path_attr = "model_so_path_" + suffix;
+  std::string inputs_attr = "inputs_" + suffix;
+  std::string outputs_attr = "outputs_" + suffix;
+  const auto& model_so_path = data_loader.attr(path_attr.c_str()).toStringRef();
+  const auto& ref_output_tensors =
+      data_loader.attr(outputs_attr.c_str()).toTensorList().vec();
+
+  torch::inductor::AOTIModelPackageLoader runner(model_so_path);
```

Comment:
I don't think AOTIModelPackageLoader takes a .so file.

Reply:
right that's true.

```cpp
torch::inductor::AOTIModelPackageLoader(model_path)
```

so the test needs to serialize a model .pt2 package first then load it